routes/api.client.php

```diff
@@ -17,6 +17,8 @@
     // Address Route
     $router->delete('addresses/{id}', 'Addresses\AddressController@destroy');
     $router->put('addresses/{id}', 'Addresses\AddressController@update');
+    $router->post('addresses/{id}/makeDefault', 'Addresses\AddressController@makeDefault');
```

Comment:
I think these routes could be tweaked slightly, not really a fan of camel-cased slugs.

Reply:
Absolutely, this is a mistake. Thanks for pointing it out

Reply:
What do you think of:

```
$router->post('addresses/{id}/default', 'Addresses\AddressController@makeDefault');
$router->delete('addresses/{id}/default', 'Addresses\AddressController@ removeDefault');
```
My only slight concern is people thinking it's going to delete the address, or do you think the `/default` is enough to make sense?

Reply:
Ah interesting. My initial reaction is I think using `delete` might be a bit misleading, and people might think it will delete whatever the default address is.

Reply:
Hmm yeah that's a good point. 🦆